spec/controllers/evidence_hub_controller_spec.rb

```diff
@@ -0,0 +1,11 @@
+RSpec.describe EvidenceHubController do
```

Comment:
Rails 5 and RSpec prefer request specs over controller specs.

Reply:
♻️